### DISCUSSION TITLE
Locomotive sound behaviour improvements/additions

### DISCUSSION THREAD

**Discussion by EsquimaltNanaimo** [voxeltycoon/issues#485] [General]
**Is your feature request related to a problem? Please describe.**
No, it would just be cool to have this in.

**Describe the solution you'd like**
I think an option for locomotives to play a bell sound when moving in stations (whether departing, arriving or just passing through) would be neat. No other transport game has officially included this, and all North American trains have them.
Also, an option for trains to sound their horns/whistles when departing a station would be really nice - gives a more classic Tycoon vibe.
Finally, I think there should be some kind of trackside sign that triggers a locomotive to sound its horn, for level crossings, bridges and such.

**Describe alternatives you've considered**
For bells, I've tried adding the bell sound to the engine noise, just playing at a low speed - but when the locomotive is switched off, the sound pitches down with the rest and sounds very odd. Plus it's rather unreliable, as more powerful engines reach the bell speed faster and end up barely ringing it.

**Additional context**
Train bell in action:
https://www.youtube.com/watch?v=BKWYz18V5DQ&t=47

**Comment by andrewpey** [comment]
![image](https://user-images.githubusercontent.com/1748681/58345147-5d5f5600-7e60-11e9-8e2c-85c161b40827.png)
